ztp/policygenerator/policyGen/policyBuilder.go

```diff
@@ -49,7 +56,7 @@ func (pbuilder *PolicyBuilder) Build(policyGenTemp utils.PolicyGenTemplate) (map
 					name = fmt.Sprintf("Multiple-%s", strings.TrimSuffix(sFile.FileName, filepath.Ext(sFile.FileName)))
 				} else {
 					// Single-resource yaml - construct a filename based on the contents
-					resource := resources[0]
+					resource := resources[0].builtCR
```

Comment:
The intent is to bind together the sourceFile data with the built object where it is constructed (which is in getCustomResources). Further down this array of resources gets passed into the createAcmPolicy and appendAcmPolicy methods which need a generatedCR object (they need the complianceType out of the sourceFile).

Reply:
no, create the generatedCR struct after the getCustomResources func return. The `for loop `  in the comment below has been created because of changing the getCustomResources func return AND there is no need for that when we generate the CRs without wrapping it in policy. Before call the createAcmPolicy or appendAcmPolicy funcs create the struct and pass it.